views/surveys/pullAndParseSurveys.py

```diff
@@ -4,12 +4,13 @@
 import requests
 from bs4 import BeautifulSoup
 
-database = "Actual"
-username = "postgres"
-password = "password"
-address = "10.11.12.24"
-googleEmail = "pokcpcapep@gmail.com"
-googlePassword = "Marist1234"
+# config.ini information is passed in as arguments from php shell_exec
```

Comment:
Hides the raw text email and password which as sensitive. `config.ini` is on the `.gitignore` list so it won't show up publicly on the repo.